src/Hint/Extensions.hs

```diff
@@ -341,11 +347,11 @@ used EmptyCase = hasS f
 used KindSignatures = hasT (un :: HsKind GhcPs)
 used BangPatterns = hasS isPBangPat ||^ hasS isStrictMatch
 used TemplateHaskell = hasT2' (un :: (HsBracket GhcPs, HsSplice GhcPs)) ||^ hasS f ||^ hasS isSpliceDecl
-    where
-      f :: HsBracket GhcPs -> Bool
-      f VarBr{} = True
-      f TypBr{} = True
-      f _ = False
+  where
```

Comment:
I'm inclined to switch to something like Ormulu, because the formatting in much of the code is haphazard.